version/version.go

```diff
@@ -33,8 +35,21 @@ var (
 	// GoVersion stores the version of Go used to build the binary
 	// (e.g. go1.14.2)
 	GoVersion string = runtime.Version()
+
+	promBuildInfo = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "sensu_go_build_info",
+			Help: "Sensu Go build information",
+		},
+		[]string{"version", "builddate", "buildsha", "goversion"},
+	).WithLabelValues(Version, BuildDate, BuildSHA, GoVersion)
 )
 
+func init() {
+	prometheus.MustRegister(promBuildInfo)
+	promBuildInfo.Set(1)
```

Comment:
I don't understand why `.Set(1)` in necessary; what is it for?

Reply:
It's required so the `sensu_go_build_info` metric has a value (it could have been any integer value really), otherwise it will not show up in the metrics endpoint